### PR TITLE
Fixed manifest downloads and updated tests to use latest schemas.

### LINKED ISSUE #58: Deploy dcc-dashboard-service for Commons data for HCA DSS
This may also include some work towards making the `dcc-dashboard-service` deployment for use with the HCA DSS more automated, easier and reliable.

### PR DESCRIPTION
**Pull request by jvlasblom**
Fixes #58

### CODE REVIEW

**Comment by jvlasblom** [comment]
This replaces my earlier PR https://github.com/DataBiosphere/cgp-dashboard-service/pull/77

**Comment by hannes-ucsc** [comment]
@jvlasblom, I rebased your PR and resolved the conflicts. Please make sure you reset your local branch to the remote branch before adding commits.

File: chalicelib/responseobjects/elastic_request_builder.py [hunk 1]
```diff
@@ -110,9 +111,9 @@ def create_query(filters):
         # Each iteration will AND the contents of the list
         query_list = [Q('constant_score', filter=Q(
             'terms', **{'{}__keyword'.format(
-                facet.replace(".", "__")): values['is']}))
+                facet.replace(".", "__")): values['is'] if 'is' in values else {}}))
```
Comment (hannes-ucsc) [inline chalicelib/responseobjects/elastic_request_builder.py]: `values.get('is', {})`

File: chalicelib/responseobjects/elastic_request_builder.py [hunk 2]
```diff
@@ -502,7 +506,7 @@ def transform_request(self,
                     'translation'][pagination['sort']]
             # Apply paging
             es_search = self.apply_paging(es_search, pagination)
-            # Execute ElasticSearch request
+            # Execute ElasticSearch request"
```
Comment (hannes-ucsc) [inline chalicelib/responseobjects/elastic_request_builder.py]: Remove quotes

File: chalicelib/responseobjects/hca_response_v5.py [hunk 3]
```diff
@@ -124,26 +127,49 @@ class ManifestResponse(AbstractResponse):
     """
     Class for the Manifest response. Based on the AbstractionResponse class
     """
+
+    def _translate(self, untranslated, keyname):
+        m = self.manifest_entries[keyname]
+        return [untranslated[es_name] if es_name in untranslated else "" for es_name in m.values()]
```
Comment (hannes-ucsc) [inline chalicelib/responseobjects/hca_response_v5.py]:
```
m = self.manifest_entries[keyname]
return [untranslated.get(es_name, "") for es_name in m.values()]
```

File: chalicelib/responseobjects/hca_response_v5.py [hunk 4]
```diff
@@ -124,26 +127,49 @@ class ManifestResponse(AbstractResponse):
...
+
+        headers = {'Content-Disposition': 'attachment; filename="export.tsv"',
+                   'Content-Type': 'text/tab-separated-values'}
+
+        output = StringIO()
+
+        writer = csv.writer(output, dialect='excel-tab')
 
-    def __init__(self, raw_response, manifest_entries, mapping):
+        writer.writerow(list(self.manifest_entries['bundles'].keys()) + list(self.manifest_entries['files'].keys()))
```
Comment (hannes-ucsc) [inline chalicelib/responseobjects/hca_response_v5.py:146]: I don't think it is safe to assume that every iteration over `self.manifest_entries['bundles']` and `self.manifest_entries['files']` yields the entries in the same order. Can you please make those to mappings `OrderedDict`s?

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]

**Review by hannes-ucsc — APPROVED** [review]

**Review by hannes-ucsc — APPROVED** [review]

**Pull request merged**